script/main.js

```diff
@@ -16,3 +68,111 @@ menuItems.forEach((item) => {
+    const workCardTitle = addElement('h3', workCard, 'work-card-title');
+    workCardTitle.innerHTML = work.name;
+    const workCardTags = addElement('ul', workCard, 'work-card-tags');
+    workCardTags.classList.add('nobull');
+    const workTechnologies = work.technologies;
+    workTechnologies.forEach((technology) => {
+      const workCardTag = addElement('li', workCardTags, 'work-card-tag');
+      workCardTag.innerHTML = technology;
+    });
+    const workCardButton = addElement('button', workCard, 'work-card-button');
+    workCardButton.innerHTML = 'See Project';
+
+    // open modal window
+
+    workCardButton.addEventListener('click', () => {
+      const isDisplayOrMobile = isHidden('.logo');
+
+      const modalWorkContainer = document.querySelector('.modal-work-container');
+
+      modalWorkContainer.classList.add('modal');
+      modalWorkContainer.classList.remove('hide');
+      const modalWorkItem = addElement('div', modalWorkContainer, 'work-item');
+      const modalWorkCard = addElement('div', modalWorkItem, 'work-card');
+
+      const modalFeaturedImage = addElement('img', modalWorkCard, 'work-card-image');
+      modalFeaturedImage.src = work.featuredImage;
+
+      const modalCloseIcon = addElement('a', modalWorkCard, 'close-icon');
+      modalCloseIcon.classList.add('fa');
+      modalCloseIcon.classList.add('fa-times');
+
+      if (isDisplayOrMobile) {
+        const modalWorkCardDisplay = addElement('div', modalWorkCard, 'work-card-display');
+        const modalWorkCardTitle = addElement('h3', modalWorkCardDisplay, 'work-card-title-display');
+        modalWorkCardTitle.innerHTML = work.name;
+
+        const modalWorkCardButton1 = addElement('a', modalWorkCardDisplay, 'modal-work-card-button-display');
+        modalWorkCardButton1.innerHTML = 'See Live';
+        modalWorkCardButton1.href = work.linkToLiveVersion;
+
+        const modalWorkCardButtonImg1 = addElement('img', modalWorkCardButton1, 'modal-work-card-button-img');
+        modalWorkCardButtonImg1.src = './img/see live icon.png';
+
+        const modalWorkCardButton2 = addElement('a', modalWorkCardDisplay, 'modal-work-card-button-display');
+        modalWorkCardButton2.innerHTML = 'See Source';
+        modalWorkCardButton2.href = work.linkToSource;
+
+        const modalWorkCardButtonImg2 = addElement('img', modalWorkCardButton2, 'modal-work-card-button-img');
+        modalWorkCardButtonImg2.src = './img/see source icon.png';
+      } else {
+        const modalWorkCardTitle = addElement('h3', modalWorkCard, 'work-card-title');
+        modalWorkCardTitle.innerHTML = work.name;
+      }
+
+      const modalWorkCardTags = addElement('ul', modalWorkCard, 'work-card-tags');
+      modalWorkCardTags.classList.add('nobull');
+
+      workTechnologies.forEach((technology) => {
+        const modalWorkCardTag = addElement('li', modalWorkCardTags, 'work-card-tag');
+        modalWorkCardTag.innerHTML = technology;
+      });
```

Comment:
### Your Technologies
![align-center-to-align-left](https://user-images.githubusercontent.com/93039359/196653289-2aa656c6-78ca-429c-adf6-1ee954035309.PNG)

### Figma Technologies
![align-left](https://user-images.githubusercontent.com/93039359/196653313-a861efa1-6b4d-4b32-a771-1a81cf308eca.PNG)

- [ ] You can see in the Figma template that technologies are aligned on left side in both mobile and desktop but in your design they are aligned in center. Kindly follow the Figma template to make your design consistent with your template 👍

Reply:
Thank you, it's done.